tests/src/test/java/alluxio/server/ft/journal/JournalIntegrationTest.java

```diff
@@ -130,7 +131,11 @@ public void run() {
         } catch (UnavailableException e) {
           // Ignore - this happen if the client is interrupted while trying to connect.
         } catch (IOException | AlluxioException e) {
-          mException.set(e);
+          if (e.getCause() instanceof CanceledException) {
```

Comment:
I feel like we should be able to determine if we can ignore the exception by just inspecting the type of the exception. Unrelated to this PR if we can't.

Reply:
Agreed.

Filtered CANCELLED status in the AbstractClient to throw UnavailableException.